core/actor/src/main/scala/net/liftweb/actor/LAFuture.scala

```diff
@@ -105,16 +101,18 @@ class LAFuture[T](val scheduler: LAScheduler) {
    * @return a Future that represents the function applied to the value of the future
    */
   def map[A](f: T => A): LAFuture[A] = {
-    val ret = new LAFuture[A](scheduler)
-    onComplete(v => ret.complete(v.flatMap(n => Box.tryo(f(n)))))
+    val ret = new LAFuture[A](scheduler, context)
```

Comment:
I can rename `ret` to `result` (any other suggestions appreciated) but have no idea for a better name for `f`. I looked at `Box` and `Option` and they also use just `f` as param name. 🤔